### PR TITLE
added COC

### PR DESCRIPTION
**Pull request by Manas-kashyap**
COC was given by forking @curiousLearner COC, and edited it accordingly

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,65 @@
...
+    - stalking or following;
+    - unwanted photography or recording;
+    - sustained disruption of talks or other events;
+    - intoxication at an event venue;
+    - inappropriate physical contact;
+    - unwelcome sexual attention;
+    - sexist, racist, or other exclusionary jokes;
+    - unwarranted exclusion from conference or related events based on age, gender, sexual orientation, disability, physical appearance, body size, race, religion;
+
+We want everyone to have a good time at our events. Be aware, be inclusive, be polite, be respectful and if you observe anything violating CoC, please report immediately to +91-96541 61510 or send message to @uditwapt
```
Comment (utkarsh2102) [inline README.md:62]: I think it'd be better to include the email ID, too.

File: README.md [hunk 2]
```diff
@@ -1,2 +1,65 @@
 # TechTalksCOC-
 TechTalks Code of Conduct
```
Comment (utkarsh2102) [inline README.md:2]: Remove this line. Not needed.

File: README.md [hunk 3]
```diff
@@ -1,2 +1,65 @@
 # TechTalksCOC-
```
Comment (utkarsh2102) [inline README.md:1]: Rename this to:
`# TechTalks: Code of Conduct`

**Review by utkarsh2102 — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (Manas-kashyap) [reply README.md:62]: send me email i will add it

File: README.md (hunk 2 above)
Comment (Manas-kashyap) [reply README.md:2]: Okay sure

File: README.md (hunk 3 above)
Comment (Manas-kashyap) [reply README.md:1]: okay anything else

**Pull request merged**